src/pages/timelines.md

```diff
@@ -0,0 +1,14 @@
+# Timelines
+
+On this page you will always find the current timelines for the Tractus-X/Catena-X releases. PLease be aware that in the current case the two releases 24.12 and 25.03
+will end up in one ecosystem release called **Io**
+
+PLease have a look at [Release management](/release-management), to get more information.
+
+## Release 24.12
+
+![Timeline for Release 25.03 ](@site/static/img/cx-timeline-25.03.svg)
```

Comment:
Why is here the timeline for 25.03?

Reply:
Weil wir schon an dem Release arbeiten?....

Reply:
Because @ther3sa will provide 24.12 in the afternoon https://github.com/catenax-eV/catenax-ev.github.io/pull/123#issuecomment-2378516796

Then we will have the decision to extend the development phase or not.

Reply:
Timelines are updated. Also added an overview for further upcoming releases.